### ISSUE TITLE
1.16.5 Error on joining a channel

### ISSUE THREAD

**Issue by nikitushu2** [kangarko/ChatControl#1079]
Minecraft Version: 1.16.5
MySQL: false
BungeeCord: false

```
12.06 16:02:39 [Server] INFO Dr_Mmm issued server command: /ch join local write ItzMufasa_
12.06 16:02:39 [Server] INFO ChatControlRed 10.4.6 encountered a NullPointerException! Please check your error.log and report this issue with the information in that file.
12.06 16:02:39 [Server] INFO java.lang.NullPointerException
12.06 16:02:39 [Server] INFO at org.mineacademy.chatcontrol.lib.settings.Lang.translate(Lang.java:259)
12.06 16:02:39 [Server] INFO at org.mineacademy.chatcontrol.lib.settings.Lang.of(Lang.java:244)
12.06 16:02:39 [Server] INFO at org.mineacademy.chatcontrol.lib.settings.Lang.ofScript(Lang.java:213)
12.06 16:02:39 [Server] INFO at org.mineacademy.chatcontrol.command.ChannelJoin.lambda$onChannelCommand$0(ChannelJoin.java:78)
12.06 16:02:39 [Server] INFO at org.mineacademy.chatcontrol.command.ChatControlCommands$GenericSubCommand.lambda$pollCache$0(ChatControlCommands.java:765)
12.06 16:02:39 [Server] INFO at org.mineacademy.chatcontrol.command.ChatControlCommands.handleCallbackCommand(ChatControlCommands.java:956)
12.06 16:02:39 [Server] INFO at org.mineacademy.chatcontrol.command.ChatControlCommands.access$100(ChatControlCommands.java:60)
12.06 16:02:39 [Server] INFO at org.mineacademy.chatcontrol.command.ChatControlCommands$GenericSubCommand.lambda$pollCache$2(ChatControlCommands.java:758)
12.06 16:02:39 [Server] INFO at org.mineacademy.chatcontrol.PlayerCache.lambda$poll0$5(PlayerCache.java:1103)
12.06 16:02:39 [Server] INFO at org.mineacademy.chatcontrol.lib.remain.CompRunnable$SafeRunnable.run(CompRunnable.java:68)
12.06 16:02:39 [Server] INFO at java.base/java.lang.Thread.run(Thread.java:829)
12.06 16:02:39 [Server] INFO !-----------------------------------------------------!
12.06 16:02:39 [Server] INFO Failed to execute scheduled task: java.lang.NullPointerException
```
[https://ufile.io/txwcg02a](https://ufile.io/txwcg02a) (error.log)

**Comment by ElBananaa** [comment]
> Minecraft Version: 1.16.5
> MySQL: false
> BungeeCord: false
> 
> ```
> 12.06 16:02:39 [Server] INFO Dr_Mmm issued server command: /ch join local write ItzMufasa_
> 12.06 16:02:39 [Server] INFO ChatControlRed 10.4.6 encountered a NullPointerException! Please check your error.log and report this issue with the information in that file.
> 12.06 16:02:39 [Server] INFO java.lang.NullPointerException
> 12.06 16:02:39 [Server] INFO at org.mineacademy.chatcontrol.lib.settings.Lang.translate(Lang.java:259)
> 12.06 16:02:39 [Server] INFO at org.mineacademy.chatcontrol.lib.settings.Lang.of(Lang.java:244)
> 12.06 16:02:39 [Server] INFO at org.mineacademy.chatcontrol.lib.settings.Lang.ofScript(Lang.java:213)
> 12.06 16:02:39 [Server] INFO at org.mineacademy.chatcontrol.command.ChannelJoin.lambda$onChannelCommand$0(ChannelJoin.java:78)
> 12.06 16:02:39 [Server] INFO at org.mineacademy.chatcontrol.command.ChatControlCommands$GenericSubCommand.lambda$pollCache$0(ChatControlCommands.java:765)
> 12.06 16:02:39 [Server] INFO at org.mineacademy.chatcontrol.command.ChatControlCommands.handleCallbackCommand(ChatControlCommands.java:956)
> 12.06 16:02:39 [Server] INFO at org.mineacademy.chatcontrol.command.ChatControlCommands.access$100(ChatControlCommands.java:60)
> 12.06 16:02:39 [Server] INFO at org.mineacademy.chatcontrol.command.ChatControlCommands$GenericSubCommand.lambda$pollCache$2(ChatControlCommands.java:758)
> 12.06 16:02:39 [Server] INFO at org.mineacademy.chatcontrol.PlayerCache.lambda$poll0$5(PlayerCache.java:1103)
> 12.06 16:02:39 [Server] INFO at org.mineacademy.chatcontrol.lib.remain.CompRunnable$SafeRunnable.run(CompRunnable.java:68)
> 12.06 16:02:39 [Server] INFO at java.base/java.lang.Thread.run(Thread.java:829)
> 12.06 16:02:39 [Server] INFO !-----------------------------------------------------!
> 12.06 16:02:39 [Server] INFO Failed to execute scheduled task: java.lang.NullPointerException
> ```
> 
> https://ufile.io/txwcg02a (error.log)

Hey!
Please update to the latest version, and let us know if the issue persists.

**Comment by kangarko** [comment]
Thank you, that will be fixed urgently as we roll out our 1.17 update.
